en/docs/develop/manage-repository/connect-your-own-github-repository-to-choreo.md

```diff
@@ -1,148 +1,69 @@
+Developers can collaborate via any of the following approaches:
 
-    !!! info 
-        This icon is displayed only when you complete the commit process.
+- Use the same upstream repository to create a shared component in the same organization, develop, push changes to the respective tracking branch, and eventually send a PR to the relevant branch.
+- Create their forks from the shared upstream repository, connect to individual components, commit, and send PRs to the upstream repository.
 
-Once the changes get pushed to the GitHub repository, the bottom pane of the page displays **In sync with Choreo upstream**.
-![In sync](../../assets/img/tutorials/connect-own-repo/in-sync-with-choreo-upstream.png){.cInlineImage-full}
+Once you collaborate and complete developing the REST API component, you can deploy and test it. For step-by-step instructions, see [steps 2 and 3 in the Create Your First REST API tutorial](https://wso2.com/choreo/docs/get-started/tutorials/create-your-first-rest-api/#step-2-deploy).
 
-You have now successfully synced your component source code to your own GitHub repository to allow multiple developers to develop the component in collaboration via pull requests.
+## Migrate Choreo-managed repositories to your own GitHub repositories
 
-Developers can collaborate via any of the following approaches:
-
-- Use the same upstream repository to create a shared component in the same organization, develop, push changes to the respective tracking branch, and eventually send a PR to the main branch.
-- Create their forks from the shared upstream repository, connect to individual components, commit, and send PRs to the upstream repository.
+If you previously created components on Choreo without connecting your GitHub repository,  your code resides in a Choreo-managed repository. **Choreo will remove all Choreo-managed repositories and the relevant components created using them by the 31st of January, 2023**. Choreo will send the source code to the users via mail. Users can then use this source code, store it in their GitHub repositories and create new components. 
 
-Once you collaborate and complete developing the REST API component, you can deploy it and test it. For step-by-step instructions, see [steps 2 and 3 in the Create Your First REST API tutorial](https://wso2.com/choreo/docs/get-started/tutorials/create-your-first-rest-api/#step-2-deploy).
+**If you published any connectors for these components, Choreo will continue to keep them published**. Once you create a new version of the component, you can then publish a new version of the connector. However, any applications using the original version of the connector will have to use the new connector version to use the new REST API. To do this,  a user needs to change the  ServiceUrl (REST API URL) and the auth credentials (generated from the Choreo Developer portal) used by the client application to consume the connector.
```

Comment:
Suggestion for the first sentence: "**If you have published any connectors for these components, Choreo will continue to keep them published**.

Reply:
done